### ISSUE TITLE
[Quest][Northrend] Changing the Wind's Course - corpse lost in time

### ISSUE THREAD

**Issue by Arbigast** [dalaranwow/dalaran-wow#5416]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

Quest gives item (The Lorehammer) that sends person back in time as Stormhoof to fight the North Wind and die. If Stormhoof dies during the fight you are transported back to current time, the North Wind disappears, Stormhoof does NOT drop Horn of Elemental Fury.

If you exit the (vehicle) Stormhoof you can then fight the North Wind as yourself/your character - if you die you die "back in time" and your corpse cannot be located/resurrected - after relog it spawned character in Tirisfal Glade (Deathknell) but no Spirit Healer is present at the Graveyard - Character is currently lost/unplayable/unresurrectable as a ghost in Deathknell
**Expected behaviour**: Tell us what should happen instead
Stormhoof is expected to die during the battle with the North Wind - when he dies he is to drop the object Horn of Elemental Fury which can be used by player's character to defeat the North Wind
**Steps to reproduce the problem**:

1. Accept quest (80) Changing the Wind's Course
2. Transform in Stormhoof
3. Exit vehicle (Stormhoof) and die fighting the North Wind
![wowscrnshot_042318_152213](https://user-images.githubusercontent.com/38669313/39148426-4d4db98e-470a-11e8-9bb5-2a67b6b78751.jpg)

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
Cannot reproduce this on PTR. Anyway try the unstuck tool or talk to a gamemaster which will help you to resurrect.

![GitHub Logo](http://i.imgur.com/bYkYeGQ.png)
